Docs 12572 chg term to allowlist bsj (#412)
mulesoft/docs-api-manager: modules/ROOT/pages/runtime-urls-allowlist.adoc

= Ports, IPs, and Hostnames to Allow
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
:page-aliases: runtime-urls-whitelist.adoc

> You may need to add the Anypoint Platform hostnames and ports to an allowlist in your network so that the API Gateway capabilities of the runtime can communicate with the online Anypoint Platform APIs and services that are managed by MuleSoft.

Reviewer: may > might, globally (unless actually giving permission)
```suggestion
You might need to add your network Anypoint Platform hostnames and ports to an allowlist so that the API Gateway capabilities of the runtime can communicate with the online Anypoint Platform APIs and services that are managed by MuleSoft.
```
"the runtime" = Mule or some other runtime?

Author: In context it is clear
